.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -1,34 +1,34 @@
 name: Bug report
 description: Use this template for reporting bugs. Please search existing issues first.
 labels: bug, needs-triage
+type: Bug
```

Comment:
Should automatically select the `Bug` issue type - it's a bit confusing to have labels and issue types, but the latter make it much easier to scan/browse the list of issues IMO